### PR TITLE
Add level to inventory item summary

### PR DESCRIPTION
**Pull request by GravenImageRD**
image

Addressing issue #1207

### CODE REVIEW

**Comment by stwlam** [comment]
would rather show it as not a trait tbh

**Comment by CarlosFdez** [comment]
I think as a tag is fine but showing it directly after rarity makes it look like a trait 🙃 

Right above Price maybe?

**Comment by GravenImageRD** [comment]
Sample of the new version
image

File: src/module/actor/sheet/item-summary-renderer.ts [hunk 1]
```diff
@@ -96,14 +96,12 @@ export class ItemSummaryRendererPF2e<AType extends ActorPF2e> {
...
 
-        const $priceLabel =
-            itemIsIdentifiedOrUserIsGM && item.system.stackGroup !== "coins"
-                ? ((): JQuery => {
-                      const price = item.price.value.toString();
-                      const priceLabel = game.i18n.format("PF2E.Item.Physical.PriceLabel", { price });
-                      return $(`<p>${priceLabel}</p>`);
-                  })()
-                : $();
+        let $levelPriceLabel = $();
```
Comment (CarlosFdez) [inline src/module/actor/sheet/item-summary-renderer.ts:99]: Avoid using let. If you want to make this cleaner looking than the ternary available before, do an IIFE but one layer higher (in other words, include the ` itemIsIdentifiedOrUserIsGM && item.system.stackGroup !== "coins"` as part of the inner function.

File: src/module/actor/sheet/item-summary-renderer.ts [hunk 2]
```diff
@@ -96,12 +96,12 @@ export class ItemSummaryRendererPF2e<AType extends ActorPF2e> {
...
 
-        const $priceLabel =
+        const $levelPriceLabel =
             itemIsIdentifiedOrUserIsGM && item.system.stackGroup !== "coins"
-                ? ((): JQuery => {
+                ? (() => {
                       const price = item.price.value.toString();
                       const priceLabel = game.i18n.format("PF2E.Item.Physical.PriceLabel", { price });
-                      return $(`<p>${priceLabel}</p>`);
+                      return $(`<p>${localize("Level")} ${item.level}<br/>${priceLabel}</p>`);
```
Comment (stwlam) [inline src/module/actor/sheet/item-summary-renderer.ts:104]: I don't believe "Level" is an existent localization key.

File: src/module/actor/sheet/item-summary-renderer.ts [hunk 3]
```diff
@@ -96,12 +96,12 @@ export class ItemSummaryRendererPF2e<AType extends ActorPF2e> {
...
 
-        const $priceLabel =
+        const $levelPriceLabel =
             itemIsIdentifiedOrUserIsGM && item.system.stackGroup !== "coins"
-                ? ((): JQuery => {
+                ? (() => {
                       const price = item.price.value.toString();
                       const priceLabel = game.i18n.format("PF2E.Item.Physical.PriceLabel", { price });
-                      return $(`<p>${priceLabel}</p>`);
+                      return $(`<p>${localize("LevelLabel")} ${item.level}<br/>${priceLabel}</p>`);
```
Comment (stwlam) [inline src/module/actor/sheet/item-summary-renderer.ts:104]: That also isn't going to work. Use "PF2E.LevelN" with `game.i18n.format`.

**Comment by CarlosFdez** [comment]
Wrong way to use format. I'd recommend looking for examples of game.i18n.format() in the codebase

File: src/module/actor/sheet/item-summary-renderer.ts [hunk 4]
```diff
@@ -96,12 +96,14 @@ export class ItemSummaryRendererPF2e<AType extends ActorPF2e> {
               })()
             : null;
 
-        const $priceLabel =
+        const $levelPriceLabel =
             itemIsIdentifiedOrUserIsGM && item.system.stackGroup !== "coins"
-                ? ((): JQuery => {
+                ? (() => {
```
Comment (stwlam) [inline src/module/actor/sheet/item-summary-renderer.ts:101]:
```suggestion
                ? ((): JQuery => {
```

File: src/module/actor/sheet/item-summary-renderer.ts [hunk 5]
```diff
@@ -96,12 +96,14 @@ export class ItemSummaryRendererPF2e<AType extends ActorPF2e> {
...
                       const price = item.price.value.toString();
+                      const level = item.level.toString();
```
Comment (stwlam) [inline src/module/actor/sheet/item-summary-renderer.ts:103]:
```suggestion
```

File: src/module/actor/sheet/item-summary-renderer.ts [hunk 6]
```diff
@@ -96,12 +96,14 @@ export class ItemSummaryRendererPF2e<AType extends ActorPF2e> {
...
                       const priceLabel = game.i18n.format("PF2E.Item.Physical.PriceLabel", { price });
-                      return $(`<p>${priceLabel}</p>`);
+                      const levelLabel = game.i18n.format("PF2E.LevelN", { level });
```
Comment (stwlam) [inline src/module/actor/sheet/item-summary-renderer.ts:105]:
```suggestion
                      const levelLabel = game.i18n.format("PF2E.LevelN", { level: item.level });
```

**Pull request merged**